_posts/2025-08-14-walkthrough-and-welcome-tab.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Introducing JabRef's New Walkthrough and Welcome Tab
+tags: [gsoc, walkthrough, welcome-tab]
+author: yubo
+---
+
+Hello! I'm [Yubo Cao](https://www.yubo-cao.com/), a [Google Summer of Code (GSoC) student at JabRef](https://summerofcode.withgoogle.com/myprojects/details/ae6W329p) this year. Today, I'm excited to share with you the new walkthrough and welcome tab features in JabRef that I worked on.
+
+## Background
+
+JabRef is a free, open-source reference manager that has been evolving since 2003. Over the years, our community has continuously added new features to help users manage their references more efficiently. However, some of these features can be challenging for new users to discover, and the default setting of certian features cannot be determined without prior knowledge about the user's intended usecase of JabRef. To address such challenge ([#12272](https://github.com/JabRef/jabref/issues/12272), [#12664](https://github.com/JabRef/jabref/issues/12664), and [#9461](https://github.com/JabRef/jabref/issues/9491)), walkthroughs and a welcome tab is created to help new users get started with JabRef without needing to read through the entire documentation.
+
+## What's New?
+
+With these new features, JabRef users can now:
+
+**Access Walkthroughs**: We've added walkthroughs to help new users get started with JabRef. These walkthroughs highlight the controls you need to interact with in a step-by-step manner, reducing the need to visit our [documentation](https://docs.jabref.org/) to get started with selected features. Currently, there are five walkthroughs on:
+
+1. Configure main file directory
+2. Customize the entry table
+3. Link PDF/external files to entries through the built-in full-text download, URL download, and directly drag-and-drop file
```

Comment:
```suggestion
3. Link PDF/external files to entries (e.g. through the built-in full-text download, URL download, and directly by dragging-and-dropping the file)
```
Technically, JabRef can link not just PDFs, but any filetype, so "file" or "external file" is better than just having PDF in the name. For now, I think how you wrote it is fine, but maybe you whould rename your walkthrough and update the screenshot.